packages/augur-sdk/src/state/db/LiquidityDB.ts

```diff
@@ -39,50 +51,47 @@ export class LiquidityDB extends AbstractDB {
     });
   }
 
-  async getMarketsLiquidity(marketIds?: string[]): Promise<any[]> {
-    const currentTimestamp = new BigNumber(Math.floor(Date.now() / 1000));
-    const secondsPerHour = SECONDS_IN_AN_HOUR.toNumber();
-    const mostRecentOnTheHourTimestamp = currentTimestamp.minus(currentTimestamp.mod(secondsPerHour));
-    const selectorConditions: any[] = [
-      { _id: { $ne: 'lastUpdated' } },
-      { timestamp: { $gte: mostRecentOnTheHourTimestamp.minus(SECONDS_IN_A_DAY).toNumber() } },
-    ];
-    if (marketIds) {
-      selectorConditions.push(
-        { market: { $in: marketIds } }
-      );
-    }
-    const marketsLiquidity = await this.db.find({
-      selector: {
-        $and: selectorConditions,
-      },
-    });
-    return marketsLiquidity.docs;
-  }
-
-  async recalculateLiquidity(augur: Augur, db: DB, timestamp: number): Promise<void> {
+  async recalculateLiquidity(augur: Augur, db: DB, currentTimestamp: number): Promise<void> {
+  console.log('currentTimestamp', currentTimestamp);
```

Comment:
not sure this console log is needed. nit pic

Reply:
Oops, yeah. I'll remove these.